From 28cd1d969bcc9fceb54afc2a4d9ceb9c1558265e Mon Sep 17 00:00:00 2001
From: Benjamin Mugnier <benjamin.mugnier@foss.st.com>
Date: Wed, 7 Jan 2026 13:22:05 +0100
Subject: [PATCH] defconfig: Enable vd55g1 and vd56g3 camera sensor module in
 all Pi configs

Build CONFIG_VIDEO_VD56G3 and CONFIG_VIDEO_VD55G1 as module by default.

Signed-off-by: Benjamin Mugnier <benjamin.mugnier@foss.st.com>
---
 arch/arm/configs/bcm2709_defconfig      | 2 ++
 arch/arm/configs/bcmrpi_defconfig       | 2 ++
 arch/arm64/configs/bcm2711_defconfig    | 2 ++
 arch/arm64/configs/bcm2711_rt_defconfig | 2 ++
 arch/arm64/configs/bcm2712_defconfig    | 2 ++
 5 files changed, 10 insertions(+)

--- a/arch/arm/configs/bcm2709_defconfig
+++ b/arch/arm/configs/bcm2709_defconfig
@@ -953,6 +953,8 @@ CONFIG_VIDEO_OV64A40=m
 CONFIG_VIDEO_OV7251=m
 CONFIG_VIDEO_OV7640=m
 CONFIG_VIDEO_OV9282=m
+CONFIG_VIDEO_VD55G1=m
+CONFIG_VIDEO_VD56G3=m
 CONFIG_VIDEO_AD5398=m
 CONFIG_VIDEO_AK7375=m
 CONFIG_VIDEO_BU64754=m
--- a/arch/arm/configs/bcmrpi_defconfig
+++ b/arch/arm/configs/bcmrpi_defconfig
@@ -946,6 +946,8 @@ CONFIG_VIDEO_OV64A40=m
 CONFIG_VIDEO_OV7251=m
 CONFIG_VIDEO_OV7640=m
 CONFIG_VIDEO_OV9282=m
+CONFIG_VIDEO_VD55G1=m
+CONFIG_VIDEO_VD56G3=m
 CONFIG_VIDEO_AD5398=m
 CONFIG_VIDEO_AK7375=m
 CONFIG_VIDEO_BU64754=m
--- a/arch/arm64/configs/bcm2711_defconfig
+++ b/arch/arm64/configs/bcm2711_defconfig
@@ -1047,6 +1047,8 @@ CONFIG_VIDEO_OV64A40=m
 CONFIG_VIDEO_OV7251=m
 CONFIG_VIDEO_OV7640=m
 CONFIG_VIDEO_OV9282=m
+CONFIG_VIDEO_VD55G1=m
+CONFIG_VIDEO_VD56G3=m
 CONFIG_VIDEO_AD5398=m
 CONFIG_VIDEO_AK7375=m
 CONFIG_VIDEO_BU64754=m
--- a/arch/arm64/configs/bcm2711_rt_defconfig
+++ b/arch/arm64/configs/bcm2711_rt_defconfig
@@ -1046,6 +1046,8 @@ CONFIG_VIDEO_OV64A40=m
 CONFIG_VIDEO_OV7251=m
 CONFIG_VIDEO_OV7640=m
 CONFIG_VIDEO_OV9282=m
+CONFIG_VIDEO_VD55G1=m
+CONFIG_VIDEO_VD56G3=m
 CONFIG_VIDEO_AD5398=m
 CONFIG_VIDEO_AK7375=m
 CONFIG_VIDEO_BU64754=m
--- a/arch/arm64/configs/bcm2712_defconfig
+++ b/arch/arm64/configs/bcm2712_defconfig
@@ -1049,6 +1049,8 @@ CONFIG_VIDEO_OV64A40=m
 CONFIG_VIDEO_OV7251=m
 CONFIG_VIDEO_OV7640=m
 CONFIG_VIDEO_OV9282=m
+CONFIG_VIDEO_VD55G1=m
+CONFIG_VIDEO_VD56G3=m
 CONFIG_VIDEO_AD5398=m
 CONFIG_VIDEO_AK7375=m
 CONFIG_VIDEO_BU64754=m
